admin_guide/monitoring_images.adoc

```diff
@@ -66,18 +61,35 @@ openshift/ruby      717.76MiB   3       27
 test/ruby-ex        150.65MiB   1       10
 test/django-ex      186.07MiB   1       10
 ----
-====
 
 The command displays the following information:
 
-- project and name of the `*ImageStream*`
-- size of the entire `*ImageStream*` stored in the internal
+. Project and name of the `*ImageStream*`
+. Size of the entire `*ImageStream*` stored in the internal
 xref:../install_config/registry/index.adoc#install-config-registry-overview[Red Hat Container Registry]
-- number of images this particular `*ImageStream*` is pointing to
-- number of layers `*ImageStream*` consists of
+. Number of images this particular `*ImageStream*` is pointing to
+. Number of layers `*ImageStream*` consists of
 
 
 [[monitoring-images-pruning]]
 == Pruning Images
 
 The information returned from the above commands is helpful when performing image pruning.
+
+[[viewing-resource-consumption-node]]
+== Viewing Node Resource Consumption
+
+{product-title} can display usage statistics about nodes, which provide the runtime environments for containers.
+
+To view the node resource (CPU/memory/storage) consumption:
+
+----
+$ oc adm top node -h
+
+----
+
+or
```

Comment:
These sound like they aren't equivalent, so I'd replace the entire line with "To view the pod resource consumption:"